Countries/ListViewController.swift

```diff
@@ -60,9 +60,7 @@ class ListViewController: UITableViewController {
     
     override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         guard let header = tableView.dequeueReusableHeaderFooterView(withIdentifier: CountryTableViewHeader.identifier) as? CountryTableViewHeader else { return UITableViewHeaderFooterView() }
-//        let earthImage = UIImage(named: TableConstants.headerEarthImageName)
-//        header.earthImageView.image = earthImage
-//        header.label.text = TableConstants.headerLabelText
+
```

Comment:
You could simplify your function body to:
```suggestion
return tableView.dequeueReusableHeaderFooterView(withIdentifier: CountryTableViewHeader.identifier) ?? UITableViewHeaderFooterView()
```

Reply:
Wow, looks much better, thank you